src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -46,19 +46,31 @@ class PGStringIndexerV2(StringIndexer):
 
     def _get_db_records(self, db_use_case_keys: UseCaseKeyCollection) -> Any:
         """
-        We are not querying for the use case ID because the order of
-        operations for our changes needs to be:
-        >>> 1. Change write path
+        The order of operations for our changes needs to be:
+            1. Change write path
             2. do DB backfill
-            3. Change Read path (this code)
-        We are currently at step 1
+        >>> 3. Change Read path (this code)
+        We are currently at step 3.
+        Only the performance-path Postgres table has a `use_case_id` column
+        at the moment, but this will change in the future.
```

Comment:
are we going to add the use_case_id to the release health table?

Reply:
I think the plan would be to ultimately, though I think we can get an end-to-end working on performance pipeline only first